script/deploy/facets/UpdateWhitelistManagerFacet.s.sol

```diff
@@ -0,0 +1,87 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.17;
+
+import { UpdateScriptBase } from "./utils/UpdateScriptBase.sol";
+import { stdJson } from "forge-std/StdJson.sol";
+import { WhitelistManagerFacet } from "lifi/Facets/WhitelistManagerFacet.sol";
+
+contract DeployScript is UpdateScriptBase {
+    using stdJson for string;
+
+    function run()
+        public
+        returns (address[] memory facets, bytes memory cutData)
+    {
+        return update("WhitelistManagerFacet");
+    }
+
+    function getExcludes() internal pure override returns (bytes4[] memory) {
+        bytes4[] memory excludes = new bytes4[](1);
+        excludes[0] = WhitelistManagerFacet.migrate.selector;
+
+        return excludes;
+    }
+
+    function getCallData() internal override returns (bytes memory) {
+        // Read selectors to remove from whitelistManager config
+        string memory selectorsToRemovePath = string.concat(
+            root,
+            "/config/whitelistManager.json"
+        );
+        string memory selectorsToRemoveJson = vm.readFile(
+            selectorsToRemovePath
+        );
+        string[] memory rawSelectorsToRemove = vm.parseJsonStringArray(
+            selectorsToRemoveJson,
+            ".functionSelectorsToRemove"
+        );
+        bytes4[] memory selectorsToRemove = new bytes4[](
+            rawSelectorsToRemove.length
+        );
+        for (uint256 i = 0; i < rawSelectorsToRemove.length; i++) {
+            selectorsToRemove[i] = bytes4(
+                vm.parseBytes(rawSelectorsToRemove[i])
+            );
+        }
+
+        // Read addresses to add for the current network
+        string memory addressesPath = string.concat(
+            root,
+            "/config/whitelistedAddresses.json"
+        );
+        string memory addressesJson = vm.readFile(addressesPath);
+        bytes memory rawAddresses = addressesJson.parseRaw(
+            string.concat(".", network)
+        );
+        address[] memory contractsToAdd = abi.decode(
```

Comment:
we are bypassing the isContract check here for every address.
Since we are starting fresh, I would propose that we do all we can to start clean. 

So maybe we could add a loop here that iterates over the addresses and reverts if one of them is not a contract?

Reply:
perfect, didnt see that. Thanks